src/test/java/org/apache/drill_web_test_framework/web_ui/steps/ProfilesSteps.java

```diff
@@ -17,19 +17,20 @@
  */
 package org.apache.drill_web_test_framework.web_ui.steps;
 
-import org.apache.drill_web_test_framework.web_ui.pages.BasePage;
 import org.apache.drill_web_test_framework.web_ui.pages.ProfilesPage;
 
+import static org.apache.drill_web_test_framework.web_ui.pages.BasePage.getPage;
```

Comment:
Use a method instead of a static import. Why do you need to change this class at all?

Reply:
Just wanted to improve code in adjacent area. Wont do it again